Tensile/Source/TensileTypes.h

```diff
@@ -257,7 +257,7 @@ class ProblemDims
                      - FirstStride + LastStrideB - FirstStride + idx];
     }
 
-    int numSizes() const
```

Comment:
cppcheck: Technically the member function 'ProblemDims::numSizes' can be static.